SerialPrograms/Source/PokemonHome/Programs/PokemonHome_BoxSorting.cpp

```diff
@@ -0,0 +1,505 @@
+    std::vector<std::vector<PokemonAutomation::ImageFloatBox>> slot_list{
+        {
+            PokemonAutomation::ImageFloatBox(0.06, 0.2, 0.03, 0.057),
+            PokemonAutomation::ImageFloatBox(0.132, 0.2, 0.03, 0.057),
+            PokemonAutomation::ImageFloatBox(0.204, 0.2, 0.03, 0.057),
+            PokemonAutomation::ImageFloatBox(0.276, 0.2, 0.03, 0.057),
+            PokemonAutomation::ImageFloatBox(0.348, 0.2, 0.03, 0.057),
+            PokemonAutomation::ImageFloatBox(0.42, 0.2, 0.03, 0.057),
+        },
+
+        {
+            PokemonAutomation::ImageFloatBox(0.06, 0.303, 0.03, 0.057),
+            PokemonAutomation::ImageFloatBox(0.132, 0.303, 0.03, 0.057),
+            PokemonAutomation::ImageFloatBox(0.204, 0.303, 0.03, 0.057),
+            PokemonAutomation::ImageFloatBox(0.276, 0.303, 0.03, 0.057),
+            PokemonAutomation::ImageFloatBox(0.348, 0.303, 0.03, 0.057),
+            PokemonAutomation::ImageFloatBox(0.42, 0.303, 0.03, 0.057),
+        },
+
+        {
+            PokemonAutomation::ImageFloatBox(0.06, 0.303+0.1035, 0.03, 0.057),
+            PokemonAutomation::ImageFloatBox(0.132, 0.303+0.1035, 0.03, 0.057),
+            PokemonAutomation::ImageFloatBox(0.204, 0.303+0.1035, 0.03, 0.057),
+            PokemonAutomation::ImageFloatBox(0.276, 0.303+0.1035, 0.03, 0.057),
+            PokemonAutomation::ImageFloatBox(0.348, 0.303+0.1035, 0.03, 0.057),
+            PokemonAutomation::ImageFloatBox(0.42, 0.303+0.1035, 0.03, 0.057),
+        },
+
+        {
+            PokemonAutomation::ImageFloatBox(0.06, 0.303+0.1035*2, 0.03, 0.057),
+            PokemonAutomation::ImageFloatBox(0.132, 0.303+0.1035*2, 0.03, 0.057),
+            PokemonAutomation::ImageFloatBox(0.204, 0.303+0.1035*2, 0.03, 0.057),
+            PokemonAutomation::ImageFloatBox(0.276, 0.303+0.1035*2, 0.03, 0.057),
+            PokemonAutomation::ImageFloatBox(0.348, 0.303+0.1035*2, 0.03, 0.057),
+            PokemonAutomation::ImageFloatBox(0.42, 0.303+0.1035*2, 0.03, 0.057),
+        },
+
+        {
+            PokemonAutomation::ImageFloatBox(0.06, 0.303+0.1035*3, 0.03, 0.057),
+            PokemonAutomation::ImageFloatBox(0.132, 0.303+0.1035*3, 0.03, 0.057),
+            PokemonAutomation::ImageFloatBox(0.204, 0.303+0.1035*3, 0.03, 0.057),
+            PokemonAutomation::ImageFloatBox(0.276, 0.303+0.1035*3, 0.03, 0.057),
+            PokemonAutomation::ImageFloatBox(0.348, 0.303+0.1035*3, 0.03, 0.057),
+            PokemonAutomation::ImageFloatBox(0.42, 0.303+0.1035*3, 0.03, 0.057)
+        }
+    };
+
+    uint16_t video_delay = VIDEO_DELAY; //delay for every input that need video feedback, user will be able to modify this to enhance capture card delay compatibility
+
+    uint16_t uni_delay = GAME_DELAY; //delay for non video feedback, this way I can go as fast as pokemon home can handle movement when needed
+
+    uint16_t nb_box_choosen = BOX_NUMBER; //number of box to check and sort
```

Comment:
no need to duplicate those variables, you can directly use VIDEO_DELAY, GAME_DELAY and BOX_NUMBER